dev-libs/serdisplib/serdisplib-2.01.ebuild

```diff
@@ -0,0 +1,102 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools udev
+
+DESCRIPTION="Library to drive several displays with built-in controllers or display modules"
+HOMEPAGE="http://serdisplib.sourceforge.net"
+SRC_URI="mirror://sourceforge/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="deprecated dlo experimental sdl threads tools usb"
+REQUIRED_USE="lcd_devices_acoolsdcm? ( usb )
+		lcd_devices_directgfx? ( sdl )
+		lcd_devices_glcd2usb? ( usb )
+		lcd_devices_displaylink? ( deprecated dlo )
+		lcd_devices_remote? ( experimental )"
+
+# Define the list of valid lcd devices.
+IUSE_LCD_DEVICES=( acoolsdcm ddusbt directgfx displaylink framebuffer glcd2usb
+			goldelox i2c ks0108 l4m lc7981 lh155 nokcol pcd8544
+			remote rs232 sed133x sed153x sed156x ssdoled stv8105 t6963 )
+
+# Add supported drivers from 'IUSE_LCD_DEVICES' to 'IUSE' and 'REQUIRED_USE'
+REQUIRED_USE="${REQUIRED_USE} || ("
+for lcd_driver in "${IUSE_LCD_DEVICES[@]}"; do
+	IUSE="${IUSE} lcd_devices_${lcd_driver}"
+	REQUIRED_USE="${REQUIRED_USE} lcd_devices_${lcd_driver}"
+done
+REQUIRED_USE="${REQUIRED_USE} )"
+
+RDEPEND="media-libs/gd[jpeg,png,tiff]
+	dlo? ( x11-libs/libdlo )
+	sdl? ( media-libs/libsdl )
+	usb? ( virtual/libusb:1= )
+	${DEPEND_LCD_DEVICES}"
+
+DEPEND="${RDEPEND}"
+
+src_prepare() {
+	# Apply user patches
+	eapply_user
+
+	# Run 'autotools'
+	mv configure.in configure.ac || die
+	eautoreconf
+}
+
+src_configure() {
+	# Enable all users enabled lcd devices
+	local myconf_lcd_devices
+	for lcd_device in ${!IUSE_LCD_DEVICES[@]}; do
+		if use lcd_devices_${IUSE_LCD_DEVICES[$lcd_device]}; then
+			if [ -z ${myconf_lcd_devices} ]; then
+				myconf_lcd_devices="${IUSE_LCD_DEVICES[$lcd_device]}"
+			else
+				myconf_lcd_devices="${myconf_lcd_devices},${IUSE_LCD_DEVICES[$lcd_device]}"
+			fi
+		fi
+	done
+
+	# Run 'configure'
+	econf \
+		$(use_enable deprecated) \
+		$(use_enable dlo libdlo) \
+		$(use_enable experimental) \
+		$(use_enable sdl libSDL) \
+		$(use_enable threads pthread) \
+		$(use_enable tools) \
+		$(use_enable usb libusb) \
+		--bindir="${D}"/usr/bin \
+		--includedir="${D}"/usr/include \
+		--libdir="${D}"/usr/$(get_libdir) \
+		--prefix="${D}" \
+		--sbindir="${D}"/usr/sbin \
+		--sysconfdir="${D}"/etc \
+		--with-drivers="${myconf_lcd_devices}"
+}
+
+src_install() {
+	# Set docs
+	local DOCS=( "AUTHORS" "BUGS" "DOCS" "HISTORY" "INSTALL" "PINOUTS" "README" "TODO" )
```

Comment:
You can probably remove that line, EAPI6 default should install relevant files by itself.

Reply:
Not all files are in the default, so I would like to keep it.